### PR TITLE
Functional test to test fluentd and fluentbit with fluent-logger

### PR DESCRIPTION
**Pull request by mssrivas**
### Summary
Functional test to test fluentd and fluentbit with fluent-logger

### Implementation details
Added image for fluent-logger, for here - https://github.com/fluent/fluent-logger-golang modified to work for the test case.
Added two functional test using the fluent-logger for fluentd and fluentbit

### Testing

New tests cover the changes: 

### Licensing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: agent/functional_tests/tests/functionaltests_unix_test.go [hunk 1]
```diff
@@ -1919,20 +1934,24 @@ func testFirelens(t *testing.T, firelensConfigType, secretLogOptionKey, secretLo
 	agent := RunAgent(t, agentOptions)
 	defer agent.Cleanup()
 
-	agent.RequireVersion(">=1.31.0")
-
+	agent.RequireVersion(">=1.30.0")
```
Comment (mssrivas) [inline agent/functional_tests/tests/functionaltests_unix_test.go]: Will update this before pushing

File: misc/fluent-logger/fluent-logger.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package main
```
Comment (fenxiong) [inline misc/fluent-logger/fluent-logger.go]: you should remove the binary file "misc/fluent-logger/fluent-logger" from the pr (can't add comment on the binary so commented here)

Comment (fenxiong) [reply misc/fluent-logger/fluent-logger.go]: missing license header for this file

File: agent/functional_tests/testdata/taskdefinitions/firelens-fluentd-fluent-logger/task-definition.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{
+  "family": "ecsftest-firelens-fluentd-fluent-logger",
+  "networkMode": "bridge",
```
Comment (fenxiong) [inline agent/functional_tests/testdata/taskdefinitions/firelens-fluentd-fluent-logger/task-definition.json]: i think you should add tests for both bridge and awsvpc

**Pull request merged**